lib/views/customer_details.erb

```diff
@@ -26,19 +26,19 @@
         <div class="col-6">
           <% if @customer_details.nil? %>
           <form class="pt-5" method="POST" action="/customer-details">
+            <h3 class="pt-1 pb-1"> Shipping Details </h3>
             <div class="form-group">
-              <label for="customer-name">Customer Name<span class="text-danger">*</span></label>
+              <label for="shipping-customer-name">Shipping Customer Name<span class="text-danger">*</span></label>
               <input
```

Comment:
Maybe should should change the label to just Shipping Name? (and Billing Name).

Then maybe have Company Name at the top as well.

Reply:
I mean just the label btw, not all the backend stuff